action.yml

```diff
@@ -186,7 +186,7 @@ inputs:
       Whether to run the archive step. Runs by default on deletes.
       Additional labels from the "labels" input are archived.
     required: false
-    default: ${{ github.event_name == 'delete' }}
+    default: ${{ github.event_name == 'delete' && github.event.ref_name != github.event.repository.default_branch }}
```

Comment:
What is the circumstance where we would receive a delete event for the default branch?

Reply:
Removed.

Reply:
It is an interesting case to consider, but also not obvious to me that we should special case it. Seems rare that someone would delete their default branch though, so unless there is a strong reason to keep the handling complex, I think keeping the handling simple is better.